src/oak.py

```diff
@@ -91,12 +87,24 @@ class Properties(NamedTuple):
         supports_pcd: bool = True
         """Whether the camera has a valid implementation of ``get_point_cloud``"""
 
-    family = ModelFamily("viam", "camera")
-    MODELS: ClassVar[Tuple[Model]] = (
-        Model(family, "oak"),
-        Model(family, "oak-ffc"),
-        Model(family, "oak-d"),
+    _deprecated_family = ModelFamily("viam", "camera")
+    _depr_oak_agnostic_model = Model(_deprecated_family, "oak")
+    _depr_oak_d_model = Model(_deprecated_family, "oak-d")
+    DEPRECATED_MODELS: ClassVar[Tuple[Model]] = (
```

Comment:
[q] Continue to register deprecated models for update log [here](https://github.com/viamrobotics/viam-camera-oak/pull/48/files#diff-491f430903c5a126fd9e3f25477482a4133b3c896aefac40746ac2cc802eeddcR157)?

Reply:
Added to a bullet